app/ore/project/io/ProjectFiles.scala

```diff
@@ -41,14 +51,15 @@ class ProjectFiles(val env: OreEnv) {
     * @param newName  New project name
     * @return         New path
     */
-  def renameProject(owner: String, oldName: String, newName: String): Try[Unit] = Try {
+  def renameProject(owner: String, oldName: String, newName: String): Try[Unit] = Try { //TODO
     val newProjectDir = getProjectDir(owner, newName)
     move(getProjectDir(owner, oldName), newProjectDir)
-    // Rename plugin files
-    for (channelDir <- newProjectDir.toFile.listFiles()) {
-      if (channelDir.isDirectory) {
-        for (pluginFile <- channelDir.listFiles()) {
-          move(pluginFile.toPath, getProjectDir(owner, newName).resolve(pluginFile.getName))
+
+    // Move all files
+    for (versionDir <- newProjectDir.toFile.listFiles()) {
```

Comment:
Is this needed? If we move the parent folder in the above line, we shouldn't need to move anything else.